openpype/hosts/maya/plugins/publish/validate_rig_output_ids.py

```diff
@@ -1,14 +1,22 @@
-import pymel.core as pc
+from collections import defaultdict
+
+from maya import cmds
 
 import pyblish.api
 
 import openpype.hosts.maya.api.action
+from openpype.hosts.maya.api.lib import get_id, set_id
 from openpype.pipeline.publish import (
     RepairAction,
     ValidateContentsOrder,
 )
 
 
+def get_basename(node):
```

Comment:
worth moving to lib.py?

Reply:
Dont think its worth moving atm. Its registered with the issue now, so lets leave it with that.